TestProject/ViewController.swift

```diff
@@ -12,7 +12,8 @@ class ViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         // Do any additional setup after loading the view.
-        print("Hello World Again")
+        print("Hello World Again...")
+        print("want to merge pr1")
```

Comment:
please write the statement in capital letter

Reply:
done